.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
 target/*
+/logs
```

Comment:
The `/logs` entry should not have a leading slash unless you want to ignore a specific directory at the root of the repository. If you intend to ignore all `logs` directories, it should be `logs/`.